extension/copy.js

```diff
@@ -1,40 +1,44 @@
+const COLLECTIONS_ID_PREFIX = "collection_u_";
+const SAVE_DATA_KEY = "_save-data";
+const COLLECTIONS_KEY = "___collections_";
+
 const saveData = (parent) =>
     (event) => {
         event.preventDefault();
-        localStorage.copyData = JSON.stringify(getSaveData(parent))
+        localStorage.setItem(SAVE_DATA_KEY, JSON.stringify(getSaveData(parent)));
     };
 
 const loadData = (parent) =>
     (event) => {
         event.preventDefault();
-        const copyData = JSON.parse(localStorage.copyData);
-        insertSaveData(parent, copyData);
-    }
+        const saveData = JSON.parse(localStorage.getItem(SAVE_DATA_KEY));
+        insertSaveData(parent, saveData);
+    };
 
 const getSaveData = (parent) => {
     const elements = getElementsByTags(parent, RELEVANT_TAGS);
-    return elements.reduce((acc, element) => {
-        if (element.id) {
+    return elements.reduce((saveData, element) => {
+        if (element.id && element.type !== 'hidden') {
```

Comment:
o.o so we're okay with not saving the hidden checkbox values?

Reply:
Yeah I was looking through and I found some chaos in the system. LibraryThing relies on the frontend form input for the UUID of the form. So we'd be editing another entry if we don't do this

Reply:
oh yay, cool cool makes sense